*** sshd_config.old	2017-05-30 11:23:34.328000000 +0200
--- sshd_config.new	2017-05-30 11:24:06.768000000 +0200
***************
*** 72,78 ****
  #Banner /etc/issue.net
  
  # Allow client to pass locale environment variables
! AcceptEnv LANG LC_*
  
  Subsystem sftp /usr/lib/openssh/sftp-server
  
--- 72,78 ----
  #Banner /etc/issue.net
  
  # Allow client to pass locale environment variables
! AcceptEnv LANG LC_* GIT* EDITOR
  
  Subsystem sftp /usr/lib/openssh/sftp-server
  
